tiled/catalog/migrations/versions/1cd99c02d0c7_create_index_for_fulltext_search.py

```diff
@@ -0,0 +1,40 @@
+"""Create index for fulltext search
+
+Revision ID: 1cd99c02d0c7
+Revises: a66028395cab
+Create Date: 2024-01-24 15:53:12.348880
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy import func
+
+# revision identifiers, used by Alembic.
+revision = "1cd99c02d0c7"
+down_revision = "a66028395cab"
+branch_labels = None
+depends_on = None
+
+# Make JSONB available in column
+JSONVariant = sa.JSON().with_variant(JSONB(), "postgresql")
+
+
+def upgrade():
+    connection = op.get_bind()
+    if connection.engine.dialect.name == "postgresql":
+        with op.get_context().autocommit_block():
+            # There is no sane way to perform this using op.create_index()
+            op.execute(
+                """
+                CREATE INDEX metadata_search
```

Comment:
Note to selves: if we change index name in `orm.py` as suggested below, make sure to change it here too.